README.md

```diff
@@ -29,15 +29,19 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. What is the DOM?
+Document Object Model aka building HTML in javascript, aka building html the hard way.
 
```

Comment:
When you think of the DOM think of an interface that structures the html as a tree structure where each node is an object representing an element. The DOM _represents_ the document.